File: 1-iniciante/1008/1008.cs
using System;

class URI {
    static void Main(string[] args) {
        int number, hours;
        double value;
        number = int.Parse(Console.ReadLine());
        hours = int.Parse(Console.ReadLine());
        value = double.Parse(Console.ReadLine());
        Console.WriteLine($"NUMBER = {number}");
        Console.WriteLine($"SALARY = U$ {hours * value:0.00}");
    }
}

File: 1-iniciante/1008/1008.c
#include <stdio.h>

int main() {
    int number, hours;
    double value;
    scanf("%d%d%lf", &number, &hours, &value);
    printf("NUMBER = %d\n", number);
    printf("SALARY = U$ %.2lf\n", hours * value);
    return 0;
}
